src/app/downloads/config.tsx

```diff
@@ -132,6 +137,15 @@ export const desktopLinks: DownloadLink[] = [
     isRecommended: true,
     isDeviceRelevant: (device) => device.desktop && device.mac,
   },
+  {
+    key: "flatpak",
+    shortName: "Flatpak",
+    longName: "Flatpak Application",
```

Comment:
They seem to consistently use the term "app", but it's basically the same thing (and we use all kinds of nouns here, specific to each platform/store).

Reply:
Actually, uuuhh.... I don't even know anymore...
https://docs.flatpak.org/en/latest/introduction.html#terminology

Reply:
On the same hand (bottom right corner):
![image](https://github.com/user-attachments/assets/7c05bd23-5dc4-490f-9a84-3af21e9f116c)
![image](https://github.com/user-attachments/assets/1a8e186b-6c9b-4a64-86cd-633c48f95d46)

Reply:
Changed to "App"